docs/inputs/table-group.md

```diff
@@ -17,26 +17,92 @@ Similar to an [**Input Group**](input-group), a **Table Group** allows the _**de
 The different input types appear in the menu only if an **Input Group** or a **Table Group** has already been created.
 :::
 
-The **Table Group** behaves similarly to an [**Input Group**](input-group) in that the _**developer**_ can create as many new **Table Groups** as needed, which are arranged vertically on top of each other. Similarly a **Table Group** can be renamed, deleted, or dragged to change its vertical position relative to other **Input Groups** or **Table Groups**.
+The **Table Group** behaves similarly to an [**Input Group**](input-group) in that the _**developer**_ can create as many new **Table Groups** as needed, which are arranged vertically. Similarly a **Table Group** can be renamed, deleted, or dragged to change its vertical position relative to other **Input Groups** or **Table Groups**.
 
 ## Table Group Layout
 
-The main difference between a **Table Group** and an [**Input Group**](input-group) is:
+The main differences between a **Table Group** and an [**Input Group**](input-group) is:
 
 - The inputs share the same fixed width and cannot be resized.
 - The Python variable available in the **Code** section is a two dimensional nested array (eg. `[[1,2],[3,4]]`).
-- The number of columns and the variable name are specified by the _**developer**_ at the top of the group.
-- There is no restriction on the type of input in each cell.
+- The number of columns and the variable name are specified by the _**developer**_ in the edit table popup.
+
+There are also many similarities including:
+
+- No restrictions on the type of input in each cell.
 - Cells can be left empty.
 
-For example, the following **Table Group** has three columns and two rows. The cell in the second row and second column is left blank. A slider input type is chosen for the input cell in the first row and second column while a single select input type is chosen for the input cell in the second row and third column. The array name available for the **Code** step is `arrayname`.
+For example, the following **Table Group** has three columns and 1 row. The header can be made using RichText input type and does NOT count as a row. The array name available for the **Code** step is `table`.
+
+<div style={{textAlign: 'center'}}>
+
+![img alt](/docs/inputs/table_group_numbered.png)
+
+</div>
+
+1. Drag the **Table Group** to arrange it vertically
+2. Specify a _**Title**_ that will be visible to the users
+3. Edit the **Table Group** properties (described below)
```

Comment:
instead of describe below, can do `(see [Editing a table group](inputs/table-group#editing-a-table-group))`